release-checklist.md

```diff
@@ -0,0 +1,21 @@
+# Release checklist
+- [ ] `git flow release start v19.3.0` (`v19.3.0-beta.1` if beta)
+- [ ] Check that all new changes are reflected in the changelog
+- [ ] Bump package.json version to `v19.3.0` (`v19.3.0-beta.1` if beta)
+- [ ] `nvm ls` Make sure you are on node v10+ 
+- [ ] `npm run build`
+- [ ] `npm run test`, make sure all tests are passing
+- [ ] Commit changes
+- [ ] `npm publish` or `npm publish --tag beta` for beta
+- [ ] Enter OTP
+- [ ] Make sure your `master` and `develop` branches are up-to-date
+- [ ] `git flow release finish v19.3.0` (`v19.3.0-beta.1` if beta)
+- [ ] `git push origin develop`
+- [ ] `git checkout master`
+- [ ] `git checkout -b master-test`
+- [ ] `git push origin master-test`
+- [ ] Wait for checks to finish on pushed `master-test` branch
+- [ ] `git push origin master`
+- [ ] `git branch -D master-test`
+- [ ] `git push origin :master-test`
+- [ ] Announce availability in #engineering
```

Comment:
- [ ] Announce availability in the public Slack channel: `#engineering` and on the forum here: https://forum.blockstack.org/c/releases (Include a link to the changelog)

Reply:
Addressed in update